.eslintrc.js

```diff
@@ -2,24 +2,24 @@ module.exports = {
   extends: ['algolia', 'algolia/jest', 'algolia/react', 'algolia/typescript'],
   plugins: ['react-hooks'],
   rules: {
-    'no-param-reassign': 0,
-    'import/no-extraneous-dependencies': 0,
+    'no-param-reassign': 'off',
```

Comment:
Any particular reasons this file was changed?

Reply:
I originally changed it for adding the `spellcheck` exception, and made the file consistent about 0/off then